src/js/constraint.js

```diff
@@ -1,7 +1,6 @@
 import { getClass, updateClasses } from './utils/classes';
 import { defer } from './utils/deferred';
 import { extend } from './utils/general';
-import { getBounds } from './utils/bounds';
```

Comment:
I would recommend we keep the bounds utils as utils, and import them. I think the reason you are doing `tether.bounds` is for the `this` context, correct? I would pass `this.bodyElement` as a param, rather than making these utils need to be tied to tether. Does that make sense?

Reply:
Yup! will refactor.